tests/components/spotify/test_media_browser.py

```diff
@@ -1,58 +1,95 @@
 """Test the media browser interface."""
 
+from unittest.mock import MagicMock
+
 import pytest
 from syrupy import SnapshotAssertion
 
 from homeassistant.components.spotify import DOMAIN
 from homeassistant.components.spotify.browse_media import async_browse_media
+from homeassistant.const import CONF_ID
 from homeassistant.core import HomeAssistant
-from homeassistant.setup import async_setup_component
-
-from tests.common import MockConfigEntry
 
+from . import setup_integration
+from .conftest import SCOPES
 
-async def setup_integration(hass: HomeAssistant, config_entry: MockConfigEntry) -> None:
-    """Fixture for setting up the component."""
-    config_entry.add_to_hass(hass)
-
-    await hass.config_entries.async_setup(config_entry.entry_id)
-    await hass.async_block_till_done(wait_background_tasks=True)
-    await async_setup_component(hass, DOMAIN, {})
-    await hass.async_block_till_done(wait_background_tasks=True)
+from tests.common import MockConfigEntry
 
 
+@pytest.mark.usefixtures("setup_credentials")
 async def test_browse_media_root(
     hass: HomeAssistant,
+    mock_spotify: MagicMock,
+    mock_config_entry: MockConfigEntry,
     snapshot: SnapshotAssertion,
-    spotify_setup,
+    expires_at: int,
 ) -> None:
     """Test browsing the root."""
+    await setup_integration(hass, mock_config_entry)
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        title="spotify_2",
+        unique_id="second_fake_id",
+        data={
+            CONF_ID: "second_fake_id",
+            "auth_implementation": DOMAIN,
+            "token": {
+                "access_token": "mock-access-token",
+                "refresh_token": "mock-refresh-token",
+                "expires_at": expires_at,
+                "scope": SCOPES,
+            },
+        },
+        entry_id="32oesphrnacjcf7vw5bf6odx3",
+    )
+    await setup_integration(hass, config_entry)
```

Comment:
Please add comments explaining why we add a second spotify config entry with lower case id only for this test